src/main/content/_assets/js/header.js

```diff
@@ -28,4 +26,40 @@ $(document).ready(function() {
         }
     });
 
+    if ($('#whatsNewModal').attr('href')) {
+        $("#modal-body").load($('#whatsNewModal').attr('href'), function(response, status) {
+            if (status === "success") {
+                let version = `${$('#general_title').text().replace(/[^\d.]/g, '')}`;
+                let localStorageValue = localStorage.getItem('whatsNew');
```

Comment:
no need to store it as an object, just 2 different key/value strings, it will help simplify some of this checking